JitsiConference.js

```diff
@@ -80,6 +84,7 @@ function JitsiConference(options) {
     this.connectionIsInterrupted = false;
 
 }
+JitsiConference.prototype.constructor = JitsiConference;
```

Comment:
Please have mercy :) This will bring merge conflicts with any change on the master, because the indentation has to be changed. I did that for the JingleSessionPC already and had a lot of trouble afterwards.